draft-irtf-cfrg-vdaf.md

```diff
@@ -3153,107 +3162,106 @@ struct {
+proof system and some extensions to it. The construction is specified in
+{{flp-bbcggi19-construction}}.
 
 ### Overview {#flp-bbcggi19-overview}
 
-In the proof system of {{BBCGGI19}}, validity is defined via an arithmetic
-circuit evaluated over the encoded measurement: If the circuit output is zero,
-then the measurement is deemed valid; otherwise, if the circuit output is
-non-zero, then the measurement is deemed invalid. Thus the goal of the proof
-system is merely to allow the verifier to evaluate the validity circuit over
-the measurement. For our application ({{prio3}}), this computation is
-distributed among multiple Aggregators, each of which has only a share of the
-measurement.
+Conventional zero-knowledge proof systems involve two parties:
 
-Suppose for a moment that the validity circuit `C` is affine, meaning its only
-operations are addition and multiplication-by-constant. In particular, suppose
-the circuit does not contain a multiplication gate whose operands are both
-non-constant. Then to decide if a measurement `x` is valid, each Aggregator
-could evaluate `C` on its share of `x` locally, broadcast the output share to
-its peers, then combine the output shares locally to recover `C(x)`. This is
-true because for any `SHARES`-way secret sharing of `x` it holds that
+* The prover, who holds a measurement and generates a proof of the
+  measurement's validity
 
-~~~
-C(x_shares[0] + ... + x_shares[SHARES-1]) =
-    C(x_shares[0]) + ... + C(x_shares[SHARES-1])
-~~~
+* The verifier who holds an encryption of, or commitment to, the measurement
+  and checks the proof
 
-(Note that, for this equality to hold, it may be necessary to scale any
-constants in the circuit by `SHARES`.) However this is not the case if `C` is
-not-affine (i.e., it contains at least one multiplication gate whose operands
-are non-constant). In the proof system of {{BBCGGI19}}, the proof is designed to
-allow the (distributed) verifier to compute the non-affine operations using only
-linear operations on (its share of) the measurement and proof.
+Our proof system is much the same, except the verifier is split across multiple
+Aggregators, each of which has a secret share of the measurement rather than an
+encryption of it.
 
-To make this work, the proof system is restricted to validity circuits that
-exhibit a special structure. Specifically, an arithmetic circuit with "G-gates"
-(see {{BBCGGI19}}, Definition 5.2) is composed of affine gates and any number of
-instances of a distinguished gate `G`, which may be non-affine. We will refer to
-this class of circuits as 'gadget circuits' and to `G` as the "gadget".
+Validity is defined in terms of an arithmetic circuit evaluated over the
+measurement. The inputs to this circuit are elements of a finite field that
+comprise the encoded measurement; the gates of the circuit are multiplication,
+addition, and negation operations; and the output of the circuit is a single
```

Comment:
This formulation of a circuit seems unusual, I think either 'multiplication and addition' or 'multiplication, addition, and subtraction' would be more conventional.

Reply:
Hmm, I think of addition, multiplication, and negation as analogues of OR, AND, and NOT respectively for boolean circuits, which to me is more intuitive. What about stating this analogy here, and maybe me mentioning that subtraction is negation followed by addition?

Reply:
I don't think we need to draw direct parallels to boolean circuits. Besides, if we were to project a boolean circuit into an arithmetic circuit, logical negation would probably be implemented with (1 - x), so I think the analogy doesn't hold up.

Looking at the 2019 Prio paper, it says in section 7.1 that arithmetic circuits over a ring R "may contain addition, subtraction and multiplication gates, as well as constants from R." The Wikipedia page for [Arithmetic circuit complexity](https://en.wikipedia.org/wiki/Arithmetic_circuit_complexity) says that circuits are made up of input gates, sum gates, and product gates.

Reply:
I'm not sure I aqgree, but I don't feel strongly either way. I replaced "negation" with "subtraction".